<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>534</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Interface Identification</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Messages Json de l'Arduino:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="5" colspan="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Envoie val. desiree</string>
      </property>
     </widget>
    </item>
    <item row="17" column="0" colspan="4">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Tension [-1,1]</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" rowspan="7">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="6">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="9" column="0" colspan="2">
     <widget class="QPushButton" name="pulseButton">
      <property name="text">
       <string>Commande de pulse</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" rowspan="7">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item row="16" column="3">
     <widget class="QLineEdit" name="JsonKey">
      <property name="text">
       <string>potVex</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_pathCSV">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Duree (ms)</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Enregistrement des donnees sous:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="4">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="PWMBox">
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QSpinBox" name="DurationBox">
      <property name="keyboardTracking">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>5000</number>
      </property>
      <property name="singleStep">
       <number>25</number>
      </property>
     </widget>
    </item>
    <item row="6" column="5">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Val. desiree</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" colspan="3">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Donnees brutes:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBoxPort"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Port:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4" rowspan="14">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="5" colspan="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Execice PID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
